src/components/ResultsList.tsx

```diff
@@ -10,11 +10,7 @@ import { withNavigation } from "react-navigation";
 import ResultsDetail from "./ResultsDetail";
 
 const ResultsList = ({ title, results, navigation }) => {
-  if (!results.length) {
-    return null;
-  }
-
-  return (
+  return results.length ? (
```

Comment:
I would change the condition to `results.length > 0`. This would add clarity as the conversion of numbers to boolean values my differ between languages.